base/libgit2/libgit2.jl

```diff
@@ -215,6 +215,22 @@ end
 
 Set the `url` for `remote` for the git repository `repo`.
 The default name of the remote is `"origin"`.
+
+# Examples
+
+```julia
+julia> repo_path = joinpath("test_directory", "Example")
```

Comment:
this isn't a doctest, but should it be written as if it is one in terms of showing output?